kitsune/community/templates/community/email/first_answer.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'email/base.html' %}
+
+{% block content %}
+    <p>
```

Comment:
Minor nit, but I think Kitsune templates use two-space indents. Is that not true?